# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/parsing/text-wrap-valid.html.ini
# Commit: 0fd4431f5279
# Full Hash: 0fd4431f527935234d764a167cc5a20acfc323eb
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-28 21:51:27
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - Implement text-wrap: balance for nsBlockFrame reflow. r=emilio
#   
#   A simple form of balance for short blocks, implemented by incrementally reducing
#   the effective inline-size used during line-breaking, up to the point where an
#   extra line would be created.
# ==============================================================================

diff -r 5926241957b4 -r 0fd4431f5279 testing/web-platform/meta/css/css-text/parsing/text-wrap-valid.html.ini
--- a/testing/web-platform/meta/css/css-text/parsing/text-wrap-valid.html.ini	Thu Sep 28 08:31:20 2023 +0000
+++ b/testing/web-platform/meta/css/css-text/parsing/text-wrap-valid.html.ini	Thu Sep 28 08:31:20 2023 +0000
@@ -1,36 +1,13 @@
 [text-wrap-valid.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
   [e.style['text-wrap'\] = "wrap" should set the property value]
     expected: FAIL
 
   [e.style['text-wrap'\] = "nowrap" should set the property value]
     expected: FAIL
 
-  [e.style['text-wrap'\] = "balance" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "stable" should set the property value]
-    expected: FAIL
-
   [e.style['text-wrap'\] = "pretty" should set the property value]
     expected: FAIL
 
-  [e.style['text-wrap'\] = "initial" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "inherit" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "unset" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "revert" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "revert-layer" should set the property value]
-    expected: FAIL
-
   [e.style['text-wrap'\] = "auto" should set the property value]
     expected: FAIL
 